src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/notifications/service/EmailServiceTest.java

```diff
@@ -655,6 +656,18 @@ public void givenIntervenerSolicitorRemovedEmailTemplate_whenPopulateTemplateVar
         assertEquals(PHONE_OPENING_HOURS, returnedTemplateVars.get("phoneOpeningHours"));
     }
 
+    @Test
+    public void givenJudgeReadyToReviewTemplate_whenPopulateTemplateVars_thenAddHearingDateToTemplateVars() {
+        setContestedData();
+        notificationRequest.setHearingDate("1 January 2024");
+
+        Map<String, Object> returnedTemplateVars =
+            emailService.buildTemplateVars(notificationRequest, FR_CONTESTED_DRAFT_ORDER_READY_FOR_REVIEW_JUDGE.name());
+
+        assertEquals("1 January 2024", returnedTemplateVars.get("hearingDate"));
```

Comment:
What about the other template fields that you expect to send to the notification service?

Reply:
[452b30d](https://github.com/hmcts/finrem-case-orchestration-service/pull/1943/commits/452b30d00f552dbe50b4d42bebb6fb074e290b21)